src/basic/parse-util.h

```diff
@@ -11,6 +11,26 @@
 
 typedef unsigned long loadavg_t;
 
+typedef enum Pcr {
+        PCR_CORE_FM,
+        PCR_PLAT_CONFIG,
+        PCR_EXT_CODE,
+        PCR_EXT_FM,
+        PCR_BOOT_LD,
+        PCR_GPT_PT,
+        PCR_POWER_ST,
+        PCR_SECURE_BT,
+        PCR_LINUX_KN = 9,
+        PCR_IMA_ST,
+        PCR_SYSTEMD_KN,
+        PCR_SYSTEMD_BT,
+        PCR_SYSTEMD_STB,
+        PCR_SHIM_CT,
+        PCR_LUKS_KEY,
+        PCR_ALIASES_MAX,
```

Comment:
no need for "aliases", and prefix max and invalid with a `_`

Reply:
Alright sir.
You're helping me write code I'll be proud of.

Thank you.